tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -362,108 +363,31 @@ public function theGroupShouldExist(string $groupname):bool {
 	}
 
 	/**
-	 *
-	 * @param string $groupname
-	 *
-	 * @return boolean
-	 * @throws Exception
-	 */
-	public function theGroupShouldBeAbleToBeDeleted(string $groupname):bool {
-		if (\array_key_exists($groupname, $this->createdGroups)) {
-			return $this->createdGroups[$groupname]['possibleToDelete'] ?? true;
-		}
-
-		if (\array_key_exists($groupname, $this->createdRemoteGroups)) {
-			return $this->createdRemoteGroups[$groupname]['possibleToDelete'] ?? true;
-		}
-
-		throw new Exception(
-			__METHOD__
-			. " group '$groupname' was not created by this test run"
-		);
-	}
-
-	/**
-	 * @When /^the administrator creates user "([^"]*)" using the provisioning API$/
-	 *
-	 * @param string|null $user
-	 *
-	 * @return void
-	 * @throws Exception
-	 */
-	public function adminCreatesUserUsingTheProvisioningApi(?string $user):void {
-		$this->createUser(
-			$user,
-			null,
-			null,
-			null,
-			true,
-			'api'
-		);
-		$this->pushToLastStatusCodesArrays();
-	}
-
-	/**
-	 * @Given /^user "([^"]*)" has been created with default attributes in the database user backend$/
-	 *
-	 * @param string|null $user
-	 *
-	 * @return void
-	 * @throws Exception
-	 */
-	public function userHasBeenCreatedOnDatabaseBackend(?string $user):void {
-		$this->adminCreatesUserUsingTheProvisioningApi($user);
-		$this->userShouldExist($user);
-	}
-
-	/**
-	 * @Given /^user "([^"]*)" has been created with default attributes and (tiny|small|large)\s?skeleton files$/
+	 * @Given user :user has been created with default attributes and without skeleton files
 	 *
 	 * @param string $user
-	 * @param string $skeletonType
-	 * @param boolean $skeleton
 	 *
 	 * @return void
-	 * @throws Exception
+	 * @throws Exception|GuzzleException
 	 */
 	public function userHasBeenCreatedWithDefaultAttributes(
-		string $user,
-		string $skeletonType = "",
-		bool $skeleton = true
+		string $user
 	):void {
-		if ($skeletonType === "") {
-			$skeletonType = $this->getSmallestSkeletonDirName();
-		}
-
-		$originalSkeletonPath = $this->setSkeletonDirByType($skeletonType);
-
-		try {
-			$this->createUser(
-				$user,
-				null,
-				null,
-				null,
-				true,
-				null,
-				true,
-				$skeleton
-			);
-			$this->userShouldExist($user);
-		} finally {
-			$this->setSkeletonDir($originalSkeletonPath);
-		}
+		$this->userHasBeenCreated(["userName" => $user]);
 	}
 
 	/**
-	 * @Given /^user "([^"]*)" has been created with default attributes and without skeleton files$/
+	 * @Given these users have been created without skeleton files and not initialized:
```

Comment:
It is initialize, when user is created it should be initialized with some requests by the new user. Otherwise some features wont work like setting quota. We can also cover testa cases for that

Reply:
right now the initialize user is sending req to ocs endpoint https://github.com/owncloud/ocis/blob/99c1b80fe76e6bfe8a0626c7d8469b625f6e5414/tests/acceptance/features/bootstrap/Provisioning.php#L2531
is there another PR that changes that?

Reply:
is it :eyes: . @S-Panta please, have a look and change it to graph (also consider reva)